tests/unit/Codeception/Module/MongoDbTest.php

```diff
@@ -136,4 +136,11 @@ public function testSeeElementIsObjectThrowsError()
 
         $this->module->seeElementIsObject('users', array(), 'trumpet');
     }
+
+    public function testUseDatabase()
+    {
+        $this->module->useDatabase('example');
+        $this->module->haveInCollection('stuff', array('name' => 'Ashley', 'email' => 'me@ashleyclarke.me'));
+        $this->module->seeInCollection('stuff', array('name' => 'Ashley', 'email' => 'me@ashleyclarke.me'));
+    }
```

Comment:
this should comtain some stuff to test that you don't see anything that was in previous database